profiles/pminio/compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  pminio:
+    image: minio/minio
+    environment:
+      - "MINIO_ACCESS_KEY={S3_ACCESS_KEY}"
+      - "MINIO_SECRET_KEY={S3_SECRET_KEY}"
+    ports:
+      - "{MINIO_PORT}:9000"
+      - "{MINIO_CONSOLE_PORT}:9090"
```

Comment:
Do we really need to expose the ports on localhost? I think it should be sufficient if the containers can talk to each other.